tools/export_schemas/README.md

```diff
@@ -0,0 +1,54 @@
+# BigQuery Schema Export Tool
+
+This tool exports the DDL (Data Definition Language) for all tables in a BigQuery project's region to local SQL files, organized by dataset.
+
+
+## Use Cases
+
+- **AI Context Provisioning**: Provide a snapshot of your current DDLs to AI models for high-fidelity schema context.
+- **Version Control**: Quickly dump all table schemas to Git for version tracking.
+- **Migration**: Export schemas to specific folders to assist in migrating datasets between projects or regions.
+- **Backup**: Create a snapshot of your current DDLs for disaster recovery or audit purposes.
+- **Local Development**: Analyze table structures offline without querying BigQuery repeatedly.
+- **CI/CD**: Use as a step in a CI/CD pipeline to export schemas for testing or validation.
+
+## Prerequisites
+
+- Google Cloud SDK: The bq command-line tool must be installed and authenticated.
+- Python 3.x: Installed on your local machine or Cloud Shell.
+- IAM Permissions:
+    - bigquery.jobs.create (to run the export query).
+    - bigquery.tables.get and access to INFORMATION_SCHEMA views.
```

Comment:
Nit: I think you need a few more permissions for this to work listed here, maybe suggest a pre-defined role: https://docs.cloud.google.com/bigquery/docs/information-schema-tables#required_permissions